src/main/java/spring/boot/com/sbparser/service/WordService.java

```diff
@@ -0,0 +1,9 @@
+package spring.boot.com.sbparser.service;
+
+import java.util.List;
+
+public interface WordService {
+    // void saveAll(List<Word> words);
```

Comment:
don't push commented code please

also, I can't see in the code where you save entities to db after reading file
which service/controller/listener is responsible for this? who initiates this process or is it done automatically?